src/app.js

```
@@ -1 +1,33 @@
+/* eslint-disable no-console */
 'use strict';
+
+const { copyFile } = require('fs/promises');
+
+async function app() {
+  const [source, destination] = process.argv.slice(2);
+
+  if (!source || !destination) {
+    console.error('Provide source and destination');
+
+    return;
+  }
+
+  if (source === destination) {
+    console.error('Source and destination are the same');
+
+    return;
+  }
+
+  try {
+    await copyFile(source, destination);
+    console.log(`${source} was copied to ${destination}`);
+  } catch (e) {
+    console.error(`The file could not be copied. Error: ${e}`);
+  }
+
+  process.exit(0);
```

This call to `process.exit(0)` should only be executed when the file copy is successful. A better place for it would be inside the `try` block, right after the success message is logged on line 23. In its current position, it masks failures that occur earlier.